Library/Homebrew/test/rubocops/urls_spec.rb

```diff
@@ -177,6 +177,14 @@
       "url" => "svn+http://brew.sh/foo/bar",
       "msg" => "Use of the svn+http:// scheme is deprecated, pass `:using => :svn` instead",
       "col" => 2,
+    }, {
+      "url" => "https://🫠.sh/foo/bar",
+      "msg" => "Please use the ASCII (Punycode, URL encoded) version of https://🫠.sh/foo/bar.",
+      "col" => 2,
+    }, {
+      "url" => "https://ßreｗ.sh/foo/bar",
```

Comment:
Few more test cases for unicode path & query:
* `https://example.com/ßreｗ`
* `https://example.org/?project=ßreｗ&foo=bar`

Reply:
Let's punt on different errors for now.